docs/src/@primer/gatsby-theme-doctocat/live-code-scope.js

```diff
@@ -12,6 +12,9 @@ const ReactRouterLink = ({to, ...props}) => {
   return <a href={to} {...props} />
 }
 
+// don't attempt to export this below because octicons-react's default export is deprecated
+delete octicons.default
```

Comment:
This `delete` statement is breaking the build. Going to see if this works
```suggestion
// Exclude octicons-react's default export because it's deprecated
const {default: _, ...octiconComponents} = octicons
```